upgrade/runner.py

```diff
@@ -245,3 +250,61 @@ def product_upgrade(product):
             execute(foreman_debug, 'satellite_{}'.format(sat_host),
                     host=sat_host)
             raise
+
+
+# Fabric task
+def validate_and_create_product_templates():
```

Comment:
This task should not be in runner.py, the runner.py is just for holding the upgrade runners and not helpers.
Move this to rhevm itself or tasks.